docs/maintainer/infrastructure.md

```diff
@@ -33,42 +83,84 @@ Smithy contains maintenance code for conda-forge, which is used by the `conda-sm
 
-<a id="web-services"></a>
-
 ### Web services
 
 The Heroku app providing the conda-forge web services lives in [conda-forge/conda-forge-webservices](https://github.com/conda-forge/conda-forge-webservices).
 Please note that the code logic provided by the app is in the `Smithy` repository.
 
 Bugs or suggestions regarding the service functionality should therefore be opened in `conda-forge/conda-smithy`'s [bug tracker](https://github.com/conda-forge/conda-smithy/issues).
 
-<a id="conda-forge-pinning"></a>
-
 ### conda-forge pinning
 
+Hosts the global pinnings for conda-forge, and the ongoing migrations.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-pinning) via [`conda-forge/conda-forge-pinning-feedstock`](https://github.com/conda-forge/conda-forge-pinning-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
 Package-wide dependency pins are defined in [conda_build_config.yaml](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml) in the [conda-forge/conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock).
 
 For more information on conda-forge wide package pins, please refer to [Globally pinned packages](pinning_deps.md#globally-pinned-packages).
 
 Please open a [PR](../glossary.md#term-PR) and/or an issue there, if you think a pin needs to be advanced. For more information on updating globally pinned packages, please refer to [Updating package pins](pinning_deps.md#update-pins).
 
-<a id="documentation"></a>
+### conda-forge-repodata-patches
+
+This repository creates the `repodata.json` patches used by the Anaconda.org to amend the metadata coming from the published packages.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-repodata-patches) via [`conda-forge/conda-forge-repodata-patches-feedstock`](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+### conda-forge-ci-setup
+
+This special feedstock provides a package that defines the logic to install and configure a common CI setup across providers.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-ci-setup) via [`conda-forge/conda-forge-ci-setup-feedstock`](https://github.com/conda-forge/conda-forge-ci-setup-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
 
 ### Documentation
 
-The documentation website lives in [conda-forge/conda-forge.github.io](https://github.com/conda-forge/conda-forge.github.io/), and is automatically deployed to our [online version](https://conda-forge.org/).
+The current [conda-forge.org](https://conda-forge.org) is a statically generated website published to Github Pages.
+
+- 📜 Source at [conda-forge/conda-forge.github.io](https://github.com/conda-forge/conda-forge.github.io/)
+- ⚙️ Deployed in [conda-forge.org](https://conda-forge.org/)
+
```

Comment:
We probably need to mention these three services too:

- PR Previews at [Netlify](https://app.netlify.com/sites/conda-forge-previews)
- Statistics at [GoatCounter](https://conda-forge.goatcounter.com/)
- Search backend at [Algolia](https://dashboard.algolia.com/apps/KB43FQOB7U/dashboard)
- Status JSONS at `conda-forge/status`
- Packages-to-feedstocks in `conda-forge/feedstock-outputs`